graph/undirect.go

```diff
@@ -214,6 +214,16 @@ func (e EdgePair) To() Node {
 	return nil
 }
 
+// ID returns the ID of the first non-nil edge, or 0.
+func (e EdgePair) ID() int64 {
+	if e[0] != nil {
+		return e[0].ID()
+	} else if e[1] != nil {
+		return e[1].ID()
+	}
+	return 0
```

Comment:
How would the returned `0` be differentiated from `e[0].ID()` or `e[1].ID()` if either of those edges have the *valid* `0` edge ID? Will this be an issue? Is the `EdgePair` a valid edge if it contains two `nil` edges?

The other methods on EdgePair (e.g. `From`) return `nil` when both the edges are `nil`, and `nil` can be differentiated from valid nodes which are non-nil, whereas `0` is a valid edge ID.

Reply:
The code here only deals with simple graphs. We may need to look into how some of the code that exists needs to be changed to cope with mutligraphs when they appear.